aispace2/jupyter/csp/cspjsonbridge.py

```diff
@@ -153,9 +158,12 @@ def json_to_csp(graph_json, widget_model=None):
                     scope.append(source_node['name'])
 
             if scope:
-                constraints.append(Constraint(tuple(scope), lt))
+                c = Constraint(tuple(scope), LessThan)
```

Comment:
3 `json_to_csp()` should create a CSP the same as represented by JSON. This will always create a CSP with constraint condition `LessThan`.